src/args.rs

```diff
@@ -17,40 +19,48 @@ pub struct Args {
     /// Loops sudo in the background to ensure it doesn't time out during long builds
     #[clap(long = "sudoloop", global = true)]
     pub sudoloop: bool,
+
+    /// Sets a custom AUR clone and build directory for the specified operation
+    #[clap(long, short, global = true, value_hint = ValueHint::DirPath)]
+    pub cachedir: Option<String>,
 }
 
 #[derive(Debug, Clone, Subcommand)]
 pub enum Operation {
-    /// Installs a package from either the AUR or the PacMan-defined repositories
-    #[clap(name = "install", aliases = & ["ins", "in", "i", "-S"])]
+    /// Installs a package from either the AUR or the Pacman-defined repositories
+    #[clap(bin_name = "ame", name = "install", visible_aliases = & ["-S"], aliases = & ["-Sa", "-Sr"])]
     Install(InstallArgs),
 
     /// Removes a previously installed package
-    #[clap(name = "remove", aliases = & ["rm", "rem", "r", "-R", "-Rs"])]
+    #[clap(bin_name = "ame", name = "remove", visible_aliases = & ["rm", "-Rs"])]
```

Comment:
Isn't the pacman arg to remove a package '-R'?

Reply:
Since we have `infer_subcommands = true` in the `clap` options at the top, and since no other command has an alias starting with -R, it automatically infers -R as -Rs, so keeping it as a separate alias entry is unnecessary

Reply:
Oh yeah missed that